docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst

```diff
@@ -14,24 +14,24 @@ First step is to create a new tax category.
 
     public function configureAction()
     {
-        $factory = $this->container->get('sylius.factory.tax_category');
-        $manager = $this->container->get('sylius.manager.tax_category');
+        $taxCategoryFactory = $this->container->get('sylius.factory.tax_category');
+        $taxCategoryManager = $this->container->get('sylius.manager.tax_category');
 
-        $clothing = $factory
+        $clothingTaxCategory = $taxCategoryFactory
             ->createNew()
             ->setName('Clothing')
             ->setDescription('T-Shirts and this kind of stuff.')
```

Comment:
Will it work? We don't have fluent interfaces, so `setName` won't return a tax category entity

Reply:
Good point, it's much more outdated than I thought! :)

Reply:
😄

Reply:
Fixed 🎉